catalog/ui/catalog-ui-search/src/main/webapp/component/attributes-rearrange/attributes-rearrange.view.js

```diff
@@ -25,10 +25,16 @@ var Sortable = require('sortablejs');
 var metacardDefinitions = require('component/singletons/metacard-definitions');
 
 function calculateAvailableAttributesFromSelection(selectionInterface) {
+    var types = _.union.apply(this, selectionInterface.getSelectedResults().map((result) => {
```

Comment:
[_.uniq](http://underscorejs.org/#uniq) might be a better function to use here.

Reply:
I looked into it a bit, but union still works best.

Unique is good if you end up with a single array, but I'm doing a map which produces multiple arrays.  For example: [ ['type1', 'type2'], ['type1'], ['type1']]